packages/core/src/item.ts

```diff
@@ -161,7 +161,7 @@ export const FIELD_DEFS: { [t in FieldType]: FieldDef } = {
     },
     [FieldType.Totp]: {
         type: FieldType.Totp,
-        pattern: ".*",
+        pattern: /.*/,
```

Comment:
This is actually very easy to recognise, since it's base32-encoded. See: https://github.com/google/google-authenticator/wiki/Key-Uri-Format#secret